spark/dl/src/main/scala/com/intel/analytics/bigdl/models/autoencoder/Train.scala

```diff
@@ -68,15 +68,10 @@ object Train {
         Autoencoder(classNum = 32)
       }
 
-      val state = if (param.stateSnapshot.isDefined) {
-        T.load(param.stateSnapshot.get)
+      val optimMethod = if (param.stateSnapshot.isDefined) {
+        OptimMethod.load[Float](param.stateSnapshot.get)
       } else {
-        T(
-          "learningRate" -> 0.01,
-          "weightDecay" -> 0.0005,
-          "momentum" -> 0.9,
```

Comment:
As far as I remember, Qiu xin just added momentum support into Adagrad. Need to check this.